website/Makefile

```
@@ -39,6 +40,21 @@ $(website_docs_dir):
 $(website_docs_dir)/index.md: $(mdoc_out)/index.md | $(website_docs_dir)
 	cp -R $(mdoc_out)/* $(this_dir)/docs/
 
+latest_version = $(chisel_dir)/latest-version.txt
+
+$(latest_version): $(chisel-non-website)
+	cd $(chisel_dir) && sbt emitLatestVersion
+
+$(latest_scaladoc_dir):
+	mkdir -p $@
+
+# Must touch target because it was built when the latest release was made
+$(latest_scaladoc_dir)/index.html: $(latest_version) | $(latest_scaladoc_dir)
+	$(eval VER := $(shell cat $(latest_version)))
+	$(eval URL := https://repo1.maven.org/maven2/org/chipsalliance/chisel_2.13/$(VER)/chisel_2.13-$(VER)-javadoc.jar)
+	cd $(latest_scaladoc_dir) && wget -q -O - $(URL) | tar -x
```

Can this avoid eval with more normal shell?

```suggestion
	VER=$(cat $latest_version)
	URL=https://repo1.maven.org/maven2/org/chipsalliance/chisel_2.13/${VER}/chisel_2.13-${VER}-javadoc.jar
	cd $(latest_scaladoc_dir) && wget -q -O - $URL | tar -x
```

---

Unfortunately no, you can't set variables in Makefile rules, so I followed the suggestion here: https://stackoverflow.com/a/1909390

I could maybe just do this as a shell script though. What do you think?